alerting/src/main/kotlin/org/opensearch/alerting/AlertService.kt

```diff
@@ -876,6 +877,16 @@ class AlertService(
         return searchResponse
     }
 
+    fun getCancelAfterTimeInterval(): Long {
```

Comment:
can we add this to AlertingUtils instead? https://github.com/opensearch-project/alerting/blob/main/alerting/src/main/kotlin/org/opensearch/alerting/util/AlertingUtils.kt